src/Morphir/Snowpark/Backend.elm

```diff
@@ -192,29 +191,29 @@ processFunctionMember valueName accessControlledValueDef currentPackagePath curr
 mapFunctionDefinition : Name.Name -> AccessControlled (Documented (Value.Definition () (Type ()))) ->  Path -> Path ->  GlobalDefinitionInformation () -> (Scala.MemberDecl, List GenerationIssue)
 mapFunctionDefinition functionName body currentPackagePath modulePath (typeContextInfo, functionsInfo, inlineInfo) =
     let
-       fullFunctionName =
+        fullFunctionName =
             FQName.fQName currentPackagePath modulePath functionName
-       functionClassification =
+        functionClassification =
             getFunctionClassification fullFunctionName functionsInfo
-       parameters =
+        parameters =
             processParameters body.value.value.inputTypes functionClassification typeContextInfo
-       parameterNames =
+        parameterNames =
             body.value.value.inputTypes |> List.map (\(name, _, _) -> name)
-       valueMappingContext = 
+        valueMappingContext = 
             { emptyValueMappingContext | typesContextInfo = typeContextInfo
```

Comment:
Notice that [record update syntax](https://elm-lang.org/docs/records#updating-records) is formatted differently from [record creation](https://elm-lang.org/docs/records#what-is-a-record-).

See how [elm-format](https://github.com/avh4/elm-format) changes a record update:

```elm
modEmp : Employee -> Employee
modEmp e =
    { e | name = "A" , age = e.age * 2
    }
```

Is changed to:

```elm
modEmp : Employee -> Employee
modEmp e =
    { e
        | name = "A"
        , age = e.age * 2
    }
```

Reply:
I saw the elm-formatter documentation and I see this example
![image](https://github.com/Snowflake-Labs/morphir-elm/assets/126700441/05fc57e7-fd58-4103-98dc-e2aaa68caf71)

the update happen with a space in the begining but no after the first word.
I ran the formatter in the file and I got this output:
![image](https://github.com/Snowflake-Labs/morphir-elm/assets/126700441/47f30904-64ce-4a3a-a510-0f60a7a25138)